docs/concepts/operator.md

```diff
@@ -154,9 +231,19 @@ The `gardener-resource-manager`'s [HighAvailabilityConfig webhook](resource-mana
 
 > If once set, removing `.spec.virtualCluster.controlPlane.highAvailability` again is not supported.
 
-The `virtual-garden-kube-apiserver` `Deployment` (not yet managed by the operator) is exposed via a `Service` of type `LoadBalancer` with the same name.
+The `virtual-garden-kube-apiserver` `Deployment` is exposed via a `Service` of type `LoadBalancer` with the same name.
 In the future, we might switch to exposing it via Istio, similar to how the `kube-apiservers` of shoot clusters are exposed.
 
+Like for the `Shoot` API, the version of the virtual garden cluster is controlled via `.spec.virtualCluster.kubernetes.version`.
+Similarly, specific configuration for the control plane components can be provided in the same section, e.g. via `.spec.virtualCluster.kubernetes.kubeAPIServer` for the `kube-apiserver`.
+
+For the virtual cluster, it is essential to provide a DNS domain via `.spec.virtualCluster.dns.domain`.
+**The respective DNS record is not managed by `gardener-operator` and should be manually created and pointed to the load balancer IP of the `virtual-garden-kube-apiserver` `Service`.**
+The DNS domain is used for the `server` in the kubeconfig, and for configuring the `--external-hostname` flag of the API server.
+
+It is also mandatory to provide an IPv4 CIDR for the service network of the virtual cluster via `.spec.virtualCluster.networking.services`.
```

Comment:
For now, IPv4 is mandatory since future evolvement of GOP will make according assumptions (Gardener control plane needs to be connected to the virtual cluster). I'd like to keep the IPv6 out-of-scope for now. It can probably be adapted very easily in a separate PR if needed.